datagen.ts

```diff
@@ -114,11 +115,23 @@ if (!global.wait) {
     }
 
     // Generate data
-    await dataGenerator({
-        format: options.format,
-        schema: parsedSchema,
-        iterations: options.number
-    })
+    if (options.format === 'sql') {
+        if (options.schema.split('.').pop() !== 'sql') {
+            alert({
+                type: `error`,
+                name: `Producing SQL data is only supported with SQL schema files!`,
+                msg: ``
+            });
+            process.exit(1);
+        }
+        await postgresDataGenerator({ schema: parsedSchema, iterations: options.number, initialSchema: options.schema });
```

Comment:
This also seems like it should go inside dataGenerator

Reply:
Good point! I've restructured the dataGenetrator so now we have separate generators for each destination, eg `kafkaDataGenerator`, `postgresDataGenerator` and the main `dataGenetrator` class is where the logic will reside. This should allow us to extend the datagen with other generators like MySQL if we ever needed to.

Reply:
Let me know what you think!

Reply:
👏🏻love it